src/main/java/com/dezorganizacja/bridge/view/graphical/CreditsController.java

```diff
@@ -1,10 +1,110 @@
 package com.dezorganizacja.bridge.view.graphical;
 
+import com.dezorganizacja.bridge.presenter.MainPresenter;
+import javafx.animation.PathTransition;
+import javafx.animation.Timeline;
 import javafx.event.ActionEvent;
+import javafx.geometry.Insets;
+import javafx.scene.control.Label;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.*;
+import javafx.scene.text.Font;
+import javafx.util.Duration;
+
+import static java.lang.Thread.sleep;
 
 public class CreditsController extends Controller {
 
+    public AnchorPane creditsPane;
+
     public void closeCredits(ActionEvent actionEvent) {
         presenter.setState("main");
     }
+
+    @Override
+    public void init(MainPresenter presenter) {
+        super.init(presenter);
+
+        creditsPane.setBackground(new Background(new BackgroundFill(Color.BLACK, CornerRadii.EMPTY, Insets.EMPTY)));
+
+        int duration = 10000;
+
+        VBox myCreditsBox = new VBox();
+        myCreditsBox.getChildren().add((new CreditsNodeController()).setCodePieceLabel("OPs").addAuthor("Grzegorz Świrski").addAuthor("Marek Rusinowski").accept());
+        myCreditsBox.getChildren().add((new CreditsNodeController()).setCodePieceLabel("GUI").addAuthor("Michał Ziobro").addAuthor("Bartłomiej Puget").addAuthor("Dominika Salawa").addAuthor("Maciej Wachulec").accept());
+        myCreditsBox.getChildren().add((new CreditsNodeController()).setCodePieceLabel("TUI").addAuthor("Maciej Woźniak").accept());
+
+        creditsPane.getChildren().add(myCreditsBox);
+
+        Path path = new Path();
+        path.getElements().add(new MoveTo(300,600));
+        path.getElements().add(new LineTo(300,-200));
+        PathTransition pathTransition = new PathTransition();
+        pathTransition.setDuration(Duration.millis(duration));
+        pathTransition.setPath(path);
+        pathTransition.setOrientation(PathTransition.OrientationType.NONE);
+        pathTransition.setNode(myCreditsBox);
+        pathTransition.setCycleCount(Timeline.INDEFINITE);
```

Comment:
rozumiem, że Twoim założeniem jest, żeby te napisy leciały w nieskończoność? cycle count to jest ile razy powtórzysz animację